### DISCUSSION TITLE
Is it safe for me to upgrade from ventura 13.6.4 to sonoma 14.4.1?

### DISCUSSION THREAD

**Discussion by mohddaniyaalsaifi** [ChefKissInc/NootedRed#242] [General]
Will there be any issue with the latest nooted kext?

**Comment by oceanondawave** [comment]
Yes, it will be completely safe. I'm running the latest version of Sonoma (14.4.1) with the latest build of NootedRed here and everything is so smooth. In the process of upgrading, you may face a long period time of installing on the first time booting up your device, but don't worry, it will boot up successfully. Make sure to `RESET NVRAM`  at least once to avoid any issues that may happen.

**Reply by mohddaniyaalsaifi** [reply]
thanks man

**Comment by SalmonPete** [comment]
No problem, I updated from Ventura 13.6.5 to Sonoma 14.4.1 without any issues.

**Reply by mohddaniyaalsaifi** [reply]
thank you

**Comment by VisualEhrmanntraut** [comment]
Be aware of #235.

**Comment by SalmonPete** [comment]
> Be aware of #235.

I haven't experienced any issues as mentioned in #235 with my 5700U. Selecting 'colored' wallpaper seems seamless certainly not crashing or freezing. I also think updating the Itlwm kext makes a difference to the stability of system settings. Didn't we establish before @VisualEhrmanntraut, you asked me to boot with NootedRed disabled some time ago and we established that itlwm was the issue.

**Reply by VisualEhrmanntraut** [reply]
it varies

**Comment by Zockat** [comment]
I have Ryzen 3 2200G + Vega 8 + A320M-HD Asrock + 2x4Gb RAM (Dual). Ventura 13.6.6 runs perfectly as far as I used it. However, 14.1.1 and 14.4.1. (versions I tested) after the login screen it shows a black screen and restarts. If I don't login, doesn't result in a black screen. If I use Whatevergreen.kext I don't have this problem.

**Reply by SalmonPete** [reply]
On my Ryzen 7 4700G (Renoir) — Gigabyte A520I — 32 GB RAM, the black screen and restart has only happened a couple of times for me in Sonoma 14.1.1. It might be worth checking Hackintosh forums for a solution, or ask someone to look at your EFI. As you say Ventura runs perfect, I have it installed on a GenMachine REN5000, even iMovie hasn't complained. But Sonoma is throwing up a lot of issues, not just us APU/NootedRed fans!.
